apps/storybook/src/components/ai-status-badge/index.tsx

```
@@ -0,0 +1,128 @@
+      >
+        <StatusDot status={status} size={resolvedSize} />
+        <span className={cn('font-medium', config.text)}>{displayLabel}</span>
+        {model && <span className="text-muted-foreground">&middot; {model}</span>}
+      </div>
+    );
+  }
+);
+
+AIStatusBadge.displayName = 'AIStatusBadge';
+
+export { AIStatusBadge, statusBadgeVariants };
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)